specs/Modal.spec.js

```diff
@@ -92,6 +92,12 @@ describe('Modal', function () {
     });
   });
 
+  it('handles case when child has no tabbable elements', function() {
```

Comment:
It apperas that you forgot to make an assertion in the spec. It just opens the modal, presses tab then closes it.